cms/envs/test_base.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+"""
+This config file runs the simplest dev environment using sqlite, and db-based
+sessions. Assumes structure:
+
+/envroot/
+        /db   # This is where it'll write the database file
+        /edx-platform  # The location of this repo
+        /log  # Where we're going to write log files
+"""
+
+# We intentionally define lots of variables that aren't used, and
+# want to import all variables from base settings files
+# pylint: disable=wildcard-import, unused-wildcard-import
+
+# Pylint gets confused by path.py instances, which report themselves as class
+# objects. As a result, pylint applies the wrong regex in validating names,
+# and throws spurious errors. Therefore, we disable invalid-name checking.
+# pylint: disable=invalid-name
+
+from .common import *
+import os
+from path import Path as path
+from warnings import filterwarnings, simplefilter
+from uuid import uuid4
+
+# import settings from LMS for consistent behavior with CMS
+# pylint: disable=unused-import
+from lms.envs.test import (
+    WIKI_ENABLED,
+    PLATFORM_NAME,
+    SITE_NAME,
+    DEFAULT_FILE_STORAGE,
+    MEDIA_ROOT,
+    MEDIA_URL,
+)
+
+# mongo connection settings
+MONGO_PORT_NUM = int(os.environ.get('EDXAPP_TEST_MONGO_PORT', '27017'))
+MONGO_HOST = os.environ.get('EDXAPP_TEST_MONGO_HOST', 'localhost')
+
+THIS_UUID = uuid4().hex[:5]
+
+# Nose Test Runner
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+
+_SYSTEM = 'cms'
+
+_REPORT_DIR = REPO_ROOT / 'reports' / _SYSTEM
+_REPORT_DIR.makedirs_p()
+_NOSEID_DIR = REPO_ROOT / '.testids' / _SYSTEM
+_NOSEID_DIR.makedirs_p()
+
+NOSE_ARGS = [
+    '--id-file', _NOSEID_DIR / 'noseids',
+    '--xunit-file', _REPORT_DIR / 'nosetests.xml',
+]
+
+TEST_ROOT = path('test_root')
+
+# Want static files in the same dir for running on jenkins.
+STATIC_ROOT = TEST_ROOT / "staticfiles"
+
+GITHUB_REPO_ROOT = TEST_ROOT / "data"
+DATA_DIR = TEST_ROOT / "data"
+COMMON_TEST_DATA_ROOT = COMMON_ROOT / "test" / "data"
+
+# For testing "push to lms"
+FEATURES['ENABLE_EXPORT_GIT'] = True
+GIT_REPO_EXPORT_DIR = TEST_ROOT / "export_course_repos"
+
+# Makes the tests run much faster...
+SOUTH_TESTS_MIGRATE = False  # To disable migrations and use syncdb instead
```

Comment:
I don't think so, but this file was renamed from test.py, so that different settings could be used under different scenarios. I'm happy to help clean this file up though if we both agree that this can be removed.